manim/animation/override_animation.py

```diff
@@ -0,0 +1,102 @@
+"""Decorator for Mobject-specific Animation overrides."""
+
+
+__all__ = ["override_animation", "_setup_animation_overriding"]
+
+from typing import Callable, Dict, List, Type
+
+from manim.mobject.mobject import Mobject
+from manim.utils.exceptions import MultiAnimationOverrideException
```

Comment:
use relative imports
```suggestion
from ..mobject import Mobject
from ..utils.exceptions import MultiAnimationOverrideException
```